types/office-js/index.d.ts

```diff
@@ -28595,7 +28595,7 @@ declare namespace Excel {
          * @remarks
          * [Api set: ExcelApi 1.12]
          */
-        selectedItems?: Array<string | PivotItem>;
+        selectedItems?: (string | PivotItem)[];
```

Comment:
build is failing but suggests keeping the original notation

```suggestion
        selectedItems?: Array<string | PivotItem>;
```